metals/src/main/scala/scala/meta/internal/builds/ShellRunner.scala

```diff
@@ -25,8 +25,8 @@ class ShellRunner(
     userConfig: () => UserConfiguration,
     time: Time,
     statusBar: StatusBar
-)(
-    implicit executionContext: scala.concurrent.ExecutionContext
+)(implicit
```

Comment:
Personally I'm a bit torn by this. I don't like having custom settings for scalafmt. The closer to vanilla the better. Howerver, I also dislike the way the implicit is for this. 

```
newlines.implicitParamListModifierPrefer   = before
```
I prefer that setting above, but then it adds another setting. Any preference on this? Again, I don't have a strong feeling, because my desire to not add another setting is the same as my dislike for this formatting.

Reply:
Let's leave the default in my opinion. I think it's better what was before.

Reply:
👍 sounds good.